android/assets/jsons/Civ V - Vanilla/Buildings.json

```diff
@@ -468,7 +468,9 @@
 		"hurryCostModifier": 25,
 		"requiredNearbyImprovedResources": ["Iron"],
 		"requiredTech": "Metal Casting",
-		"uniques": ["+[15]% Production when constructing [Spaceship part] units [in this city]",
+		// If spaceship parts are changed into units, the spaceship part unique should be changed to
```

Comment:
You know this is illegal json, right? Our parser is lax, but I'd still advocate moving towards conforming to the standard, not away from it. (good catch btw :wink: )

Reply:
I do, but I just confirmed to what appeared to be the coding standard. Most of the jsons are already littered with comments, and they are often even used to comment out code. Do you still think I should remove my comments?